<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContinuousPasteWidget</class>
 <widget class="QWidget" name="ContinuousPasteWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTextEdit" name="previewText"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="skipButton">
       <property name="text">
        <string>Skip</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pasteButton">
       <property name="text">
        <string>Paste</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../qrc/icons/icons.qrc">
     <normaloff>:/back.svg</normaloff>:/back.svg</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back to main page</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../qrc/icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
